apps/api/src/domain/models/digital-text/commands/translate-digital-text-page-content/digital-text-page-content-translated.event.ts

```diff
@@ -0,0 +1,10 @@
+import { CoscradEvent } from '../../../../../domain/common';
+import { BaseEvent } from '../../../shared/events/base-event.entity';
+import { TranslateDigitalTextPageContent } from './translate-digital-text-page-content.command';
+
+export type DigitalTextPageContentTranslatedPayload = TranslateDigitalTextPageContent;
+
+@CoscradEvent(`DIGITAL_TEXT_PAGE_CONTENT_TRANSLATED`)
```

Comment:
Do we prefer this or `DIGITAL_TEXT_PAGE_TRANSLATED`? Whatever we pick, naming would be consistent throughout.

Reply:
It's longer, but I like the specificity of `DIGITAL_TEXT_PAGE_CONTENT_TRANSLATED`.  No confusion and it's not user facing so should be fine.

Reply:
we will keep it